iset.mm

```diff
@@ -45982,6 +46031,26 @@ We use their notation ("onto" under the arrow).  (Contributed by NM,
       ( wf wss fss syl2anc ) ABCEHCDIBDEHFGBCDEJK $.
   $}
 
+  ${
+    fssdmd.f $e |- ( ph -> F : A --> B ) $.
+    fssdmd.d $e |- ( ph -> D C_ dom F ) $.
+    $( Expressing that a class is a subclass of the domain of a function
+       expressed in maps-to notation, deduction form.  (Contributed by AV,
+       21-Aug-2022.) $)
+    fssdmd $p |- ( ph -> D C_ A ) $=
+      ( cdm fdmd sseqtrd ) ADEHBGABCEFIJ $.
+  $}
+
+  ${
+    fssdm.d $e |- D C_ dom F $.
+    fssdm.f $e |- ( ph -> F : A --> B ) $.
+    $( Expressing that a class is a subclass of the domain of a function
+       expressed in maps-to notation, semi-deduction form.  (Contributed by AV,
+       21-Aug-2022.) $)
+    fssdm $p |- ( ph -> D C_ A ) $=
+      ( cdm wf wceq fdm syl syl5sseq ) AEHZDBFABCEINBJGBCEKLM $.
```

Comment:
```
      ( cdm wss a1i fssdmd ) ABCDEGDEHIAFJK $.
```
and same in set.mm (probably no need for credit tag; letting @avekens know)

Reply:
The minimizer found a proof which is one byte shorter (in the compressed proof):

```
( cdm fdmd syl5sseq ) AEHDBFABCEGIJ $.
```

Thus I've updated this pull request with that one.